index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/8b3d2372ca.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,500;0,700;1,400&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+
+    <div class="container">
+        <div class="navbar">
+            <div class="logo">
+                G
+            </div>
+            <div class="menu">
+                <i class="fas fa-bars"></i>
+            </div>
+        </div>
+        <div class="section">
+            <div class="hero">
+                <p>Hey! <br>
```

Comment:
- [ ] The main title is an `<h1>` tag. Please kindly fix that. 
- [ ] The content should be aligned with flexbox.
- [ ] The section should have margins (top, bottom, left, right) defined in the design.
- [ ] The section should use the font and colors (background and font) defined for the template. Please kindly fix that.